src/garage/torch/modules/tanh_gaussian_mlp_module.py

```diff
@@ -0,0 +1,208 @@
+"""GaussianMLPModule."""
```

Comment:
Is there no way to implement this as a subclass or wrapper on GaussianMLPModule(s)?